### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by alivane**
Assignment 4 - Alisson Gutierrez 
Task 6 y Task 7

### CODE REVIEW

File: Assignment4/alivane-23A289/Task07.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    print(s)
+
+print("QUERY SPARQL")
+# Visualize the results
+q1 = prepareQuery('''
+    SELECT ?s
+    WHERE {
+        ?s RDFS:subClassOf ns:LivingThing .
+    }
+    ''',
```
Comment (anaigmo) [inline Assignment4/alivane-23A289/Task07.py:49]: missing recursive subclass of relationships

File: Assignment4/alivane-23A289/Task07.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  print(s)
+# Visualize the results
+
+print("QUERY SPARQL")
+
+q4 = prepareQuery('''
+  SELECT ?s
+  WHERE {
+    ?s FOAF:knows ns:RockySmith .
+  }
```
Comment (anaigmo) [inline Assignment4/alivane-23A289/Task07.py:186]: list name, not iri

File: Assignment4/alivane-23A289/Task07.py [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+print("QUERY SPARQL")
+q5 = prepareQuery('''
+        SELECT ?s
+        WHERE {
+            ?s FOAF:knows ?o1 .
+            ?s FOAF:knows ?o2 .
+            FILTER (?o1 != ?o2)
+        }
+        GROUP BY ?s
+        HAVING (COUNT(?o1) > 1)
```
Comment (anaigmo) [inline Assignment4/alivane-23A289/Task07.py:226]: redundant with filter :)

File: Assignment4/alivane-23A289/Task07.py (hunk 1 above)
Comment (alivane) [reply Assignment4/alivane-23A289/Task07.py]: It is solved :)

File: Assignment4/alivane-23A289/Task07.py (hunk 2 above)
Comment (alivane) [reply Assignment4/alivane-23A289/Task07.py]: It is solved

File: Assignment4/alivane-23A289/Task07.py (hunk 3 above)
Comment (alivane) [reply Assignment4/alivane-23A289/Task07.py]: It is solved

**Pull request merged**